docs/docs/identity-providers/readme.md

```diff
@@ -8,23 +8,14 @@ description: >-
 
 # Identity Provider Configuration
 
-This article describes how to configure Pomerium to use a third-party identity service for single-sign-on.
+Pomerium helps provide a zero-trust security model by verifying that every request to your upstream applications are from an authenticated and authorized user. User and group information is sourced from your Identity Provider (**IdP**). Pomerium communicates with IdPs following the [OpenID Connect][openid connect] (**OIDC**) standard for user identity, and a service account and/or API token for group/directory information. The steps for integrating Pomerium with an IdP are specific to each provider, but they generally share the same base requirements:
 
-There are a few configuration steps required for identity provider integration. Most providers support [OpenID Connect] which provides a standardized identity and authentication interface.
+- A **[Redirect URL](https://www.oauth.com/oauth2-servers/redirect-uris/)** pointing back to Pomerium. For example, `https://${authenticate_service_url}/oauth2/callback`.
```

Comment:
```suggestion
- A **[Redirect URL](https://www.oauth.com/oauth2-servers/redirect-uris/)** pointing back to Pomerium. For example, `https://${authenticate_service_url}/oauth2/callback`.
```
It's worth highlighting that the path `/oauth2/callback` is hardcoded. And that this URL is set within the identity provider, not Pomerium.